src/test/java/core/basesyntax/service/FruitTransactionParserImplTest.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.FruitTransactionParserImpl;
+import core.basesyntax.service.impl.OperationValidatorImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTransactionParserImplTest {
+    private static List<String> testData;
+    private static FruitTransactionParser fruitTransactionParser;
+
+    @BeforeClass
+    public static void setUp() {
+        testData = new ArrayList<>();
+        OperationValidator operationValidator = new OperationValidatorImpl();
+        fruitTransactionParser = new FruitTransactionParserImpl(operationValidator);
+    }
+
+    @Before
+    public void prepareData() {
+        testData.clear();
+        testData.add("type,fruit,quantity");
+    }
+
+    @Test
+    public void parseTransaction_getFruit_ok() {
+        testData.add("b,banana,20");
+        List<FruitTransaction> expected = new ArrayList<>();
+        expected.add(new FruitTransaction("b", new Fruit("banana"), 20));
+        List<FruitTransaction> actual = fruitTransactionParser.parseFruitTransactions(testData);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void parsTransaction_wrongFruit_notOk() {
+        testData.add("b,error,0");
+        List<FruitTransaction> expected = new ArrayList<>();
+        expected.add(new FruitTransaction("b", new Fruit("banana"), 20));
+        List<FruitTransaction> actual = fruitTransactionParser.parseFruitTransactions(testData);
+        Assert.assertNotEquals(expected.toString(), actual.toString());

```

Comment:
check previous comment 
we should use `Assert.assertEquals(someExpectedValue, actual);`
because `assertNotEquals` isn't so strict, we can pass there anything except returned value and our test will pass
we should test this better